doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -390,6 +378,41 @@ To use nRF21540 in SPI or mixed mode, complete the following steps:
    Replace the SPI bus according to your hardware design.
 
 #. Create alternative pinctrl entries for SPI3 and replace the ``pinctrl-N`` and ``pinctrl-names`` properties.
+#. On nRF53 devices, the devicetree nodes described above must be added to the network core.
+   For the application core, a GPIO forwarder node must be added to the devicetree file:
```

Comment:
This applies to the GPIO only section further up as well.

Reply:
It doesn't apply - for backwards compatibility purpose, nRF21540 GPIO implementation uses a different pin forwarding mechanism than nRF21540 GPIO+SPI implementation. The GPIO forwarder node is only needed when nRF21540 GPIO+SPI implementation is used.

Reply:
Out of curiosity, where is that defined?

Reply:
[Binding](https://github.com/nrfconnect/sdk-zephyr/blob/5c19b37dd82cb25c02da195af2666e75369b7fad/dts/bindings/gpio/nordic%2Cnrf-gpio-forwarder.yaml) and its [example usage](https://github.com/nrfconnect/sdk-zephyr/blob/5c19b37dd82cb25c02da195af2666e75369b7fad/boards/nordic/nrf5340dk/nrf5340_cpuapp_common.dtsi#L113-L119) for nRF5340 DK.

Reply:
Discussed f2f. Agreed to keep the GPIO only section as it is, since it is correct for the non-generic GPIO forwarder and is backwards compatible.